packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/ImageEdit.ts

```diff
@@ -407,11 +372,31 @@ export default class ImageEdit implements EditorPlugin {
         htmlData.forEach(data => {
             const element = createElement(data, this.image.ownerDocument);
             if (element) {
-                wrapper.appendChild(element);
+                this.wrapper.appendChild(element);
             }
         });
 
-        this.insertImageWrapper(this.image, wrapper);
+        this.insertImageWrapper(this.image, this.wrapper);
+    }
+
+    private removeVisibilityCssTag() {
+        const doc = this.editor.getDocument();
+        const styleTag = doc.getElementById(STYLE_IMAGE) as HTMLStyleElement;
+        if (styleTag) {
+            doc.head.removeChild(styleTag);
+        }
+    }
+
+    private addVisibilityCssTag() {
+        const doc = this.editor.getDocument();
+        let styleTag = doc.getElementById(STYLE_IMAGE) as HTMLStyleElement;
+        if (!styleTag) {
+            const cssRule = `.${STYLE_IMAGE} {visibility: hidden}`;
+            styleTag = doc.createElement('style');
+            styleTag.id = STYLE_IMAGE;
```

Comment:
Actually I think you can hold a reference to this style element so no id is needed at all.